### PR TITLE
Added a new trigger on node maintenance task

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Add a security event trigger on node maintenance task

# Impacts
pfmon

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* New security event in node maintenance task

### CODE REVIEW

File: lib/pf/node.pm [hunk 1]
```diff
@@ -766,7 +766,17 @@ sub nodes_maintenance {
...
         my $currentMac = $row->{mac};
         pf::dal->set_tenant($row->{tenant_id});
+
+        my $apiclient = pf::client::getClient;
+        my %violation_data = (
+            'mac'   => $currentMac,
+            'tid'   => 'node_maintenance',
+            'type'  => 'internal',
+        );
+        $apiclient->call('trigger_violation', %violation_data);
```
Comment (nqb) [inline lib/pf/node.pm]: `trigger_violation` and `%violation_data` need to be renamed to `security_event`.

**Review by nqb — CHANGES_REQUESTED** [review]

**Comment by julsemaan** [comment]
@nqb, take another look and merge

File: lib/pf/node.pm [hunk 2]
```diff
@@ -777,7 +777,17 @@ sub nodes_maintenance {
...
         my $currentMac = $row->{mac};
         pf::dal->set_tenant($row->{tenant_id});
+
+        my $apiclient = pf::client::getClient;
+        my %security_event = (
+            'mac'   => $currentMac,
+            'tid'   => 'node_maintenance',
+            'type'  => 'internal',
+        );
+        $apiclient->call('security_event', %security_event);
```
Comment (nqb) [inline lib/pf/node.pm]:
```suggestion
        $apiclient->notify('trigger_security_event', %security_event);
```

Comment (nqb) [reply lib/pf/node.pm]: Make following change and I will merge.

**Comment by fdurand** [comment]
BUMP

**Pull request merged**